clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesPatchManifestDescription.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.netflix.spinnaker.clouddriver.kubernetes.description.KubernetesAtomicOperationDescription;
+import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.KubernetesCoordinates;
+import com.netflix.spinnaker.kork.artifacts.model.Artifact;
+import java.util.List;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.apache.commons.lang3.tuple.Pair;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class KubernetesPatchManifestDescription extends KubernetesAtomicOperationDescription {
+  private String manifestName;
+  private String location;
+  // This will only be a portion of a full manifest so calls to some required fields can fail.
```

Comment:
Any reason not to just use a `Map<String, Object>` in place of `KubernetesManifest`?

Reply:
Also, I don't think "manifest" is necessarily the right name... not sure what's better... maybe `patchRequest`? or `patchBody`?

Reply:
Using `KubernetesManifest` makes it very easy to use `ArtifactReplacer`. Agree that `patchBody` is a better name